designs/virtualization/4.x/expose-guest-data/expose guest data.md

```diff
@@ -0,0 +1,55 @@
+# Expose Guest Data
+
+When the Guest agent is installed on a VM some additional valuable data can be collected then surfaced in the UI.
+When the Guest agent isn’t installed we will inform the users about that.
+
+We can report that in the VM’s Overview tab under the Status card and provide a message: “This virtual machine does not have a guest agent installed. Some metrics,and management features will not be available or accurate” and add an external link to view the details about the installation procedure of the Guest agent, if the users are interested to do so. The link will lead the users to the instructions on how to install the GA. Linux users will need to download and install the GA online, while Windows users will get the installation on a CD ROM.
+After the installation they will need to restart the VM.
+
+![Notifying users that GA isn't installed](img/vm-overview-tab.png)
+
+## After installing the Guest agent
+
+Some of the data that is collected via the Guest agent can be surfaced in the VM’s Overview tab under the Details card, as well as on the Details tab:
+
+- Hostname
+- Guest OS version - what is the actual OS version that is installed on the VM
+- Time zone
+- Users list - active users that are currently logged in to the VM
+
+Host name, Guest OS version and time zone will be added to the details card in the Overview tab.
+
+### VM’s Overview tab > Details card
+
+![Additional data when GA is installed](img/vm-dashboard-w-ga-installed.png)
+
+Clicking the number of logged in users will lead to a table with all the relevant info about these logged in users.
```

Comment:
Where does this table live? Is it a popover? Can we provide a design for this?

Reply:
@lizsurette The logged in users table lives at the bottom of the VM's details page.

Reply:
@yfrimanm Ah yes, I see now :) Thanks!